src/trans/gpu/CMakeLists.txt

```diff
@@ -61,6 +61,7 @@ ecbuild_add_library(
                        $<${HAVE_CUTLASS}:USE_CUTLASS>
                        $<${HAVE_CUTLASS_3XTF32}:USE_CUTLASS_3XTF32>
                        $<${HAVE_GPU_GRAPHS_GEMM}:USE_GRAPHS_GEMM>
+		       $<${HAVE_GPU_GRAPHS_FFT}:USE_GRAPHS_FFT>
```

Comment:
```suggestion
                       $<${HAVE_GPU_GRAPHS_FFT}:USE_GRAPHS_FFT>
```

Reply:
Sorry for being pedantic :) I just replaced the tab characters with spaces.

Reply:
It's good to be pedantic, and I am always for replacing characters with spaces :D